src/toro.cr

```diff
@@ -134,6 +147,16 @@ module Toro
 
     abstract def routes
 
+    def json(response : T, status_code : Int? = nil)
```

Comment:
Can you put this commit in a separate pull request? That way I can merge only the halt changes.